code/datums/wires/wires.dm

```diff
@@ -121,15 +121,23 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 			var/obj/item/I = L.get_active_hand()
 			holder.add_hiddenprint(L)
 			if(href_list["cut"]) // Toggles the cut/mend status
-				if(iswirecutter(I))
+				var/switchtool_deployed
```

Comment:
I was going to do this a very different way, with a lot more unnecessary code, but shadowmech said to do it this way so it would cut down the size and complexity, so I'm just going with it. I don't fully understand everything but it works.